specification/security/resource-manager/Microsoft.Security/stable/2020-01-01/applicationWhitelistings.json

```diff
@@ -553,10 +553,6 @@
       "description": "Include the policy rules",
       "type": "boolean",
       "required": false,
-      "enum": [
-        false,
-        true
-      ],
```

Comment:
type 'boolean' only has two kinds of valuses : 'true' and 'false'. So there is no need to define it redundantly.

Reply:
@ArcturusZhang @weidongxu-microsoft  @dw511214992 @allenjzhang 
We delete redundant enum definition for 'boolean', please check whether it influence your language.